beckFilter/challenges-08.test.js

```diff
@@ -0,0 +1,309 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named oddValues that, given an array of integers as input,
+// uses filter to return an array containing only the odd integers.
+//
+// For example, oddValues([1,2,3]) returns [1,3].
+// ------------------------------------------------------------------------------------------------
+
+const oddValues = (input) => {
+  // Solution code here...
+  return input.filter( digit => digit % 2 === 1);
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named filterStringsWithVowels that, given an array of strings as input,
+// uses filter to return an array with only words that contain vowels.
+//
+// The callback function to filter should include a regular expression pattern.
+//
+// For example, filterStringsWithVowels('gregor','hound','xyz') returns ['gregor', 'hound'].
+// ------------------------------------------------------------------------------------------------
+
+
+const filterStringsWithVowels = (input) => {
+  // Solution code here...
+  return input.filter( word => word.match(/[aeiou]/g));
```

Comment:
Would be better to use `/regex/.test(word)` here--that one explicitly returns a boolean, while `.match` is only working because of truthy/falsyness.